src/features/party/PartiesProvider.tsx

```diff
@@ -51,6 +51,25 @@ export function useCurrentPartyQueryDef(enabled: boolean) {
   };
 }
 
+export function useCurrentPartyRolesQueryDef(enabled: boolean) {
+  const { fetchRoles } = useAppQueries();
```

Comment:
Prøver å refaktorere oss ut fra `useAppQueries` siden det bare er statiske funksjoner og dermed ikke trenger å være i en context. Du kan bare importere funksjonen direkte fra fila den er i (eller til og med ha den i denne fila, eller nærmere i hvert fall for collocation?). Hvis du lar den være i appqueries må du også huske på å mocke den riktig i tester og legge den til i `IgnoredQueries` i `src/queries/types.ts` for at den ikke skal være påkrevd i test-setup. 😊

Reply:
done :)